packages/sdk-metrics/src/aggregator/exponential-histogram/mapping/LogarithmMapping.ts

```diff
@@ -0,0 +1,138 @@
+/*
+ * Copyright The OpenTelemetry Authors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+import * as ieee754 from './ieee754';
+import * as util from '../util';
+import { Mapping, MappingError } from './types';
+
+/**
+ * LogarithmMapping implements exponential mapping functions for scale > 0.
+ * For scales <= 0 the exponent mapping should be used.
+ */
+export class LogarithmMapping implements Mapping {
+  static readonly MIN_SCALE = 1;
+  static readonly MAX_SCALE = 20;
```

Comment:
Are these public for a reason? Typically we do constants as `const` outside of the class so that they can be more easily minified. `LogarithmMapping.MAX_SCALE` cannot be minified, but the const variable `MAX_SCALE` can be minified to a single letter (or subtituted and compiled away for very aggressive minifiers). But if they need to be exported then they can't be minified anyway.

Reply:
The only place these are referenced are in tests. I can hard code the values since it will have a minification benefit. While I'm at it, does it make sense, in terms of minification, or style, to get rid of the statics altogether? Instead of having `LogarithmMapping.get`, I could export a `getLogarithmMapping` function and move the prebuilt mappings outside of the class.

Reply:
Yes that would help minification but it is only worth it if it doesn't significantly impact the readability of the code. Up to you.